src/sql/azdata.proposed.d.ts

```diff
@@ -741,6 +745,15 @@ declare module 'azdata' {
 			viewModel: DesignerViewModel;
 		}
 
+		/**
+		 * Table node sub type (Temporal/GraphEdge/GraphNode)
```

Comment:
Remove the type names - the enum defines these already so it's just duplicating that